packages/web-pkg/src/composables/spaces/useGetMatchingSpace.ts

```diff
@@ -0,0 +1,47 @@
+import { useConfigurationManager, useStore } from 'web-pkg/src/composables'
+import { Resource, SpaceResource } from 'web-client'
+import {
+  buildPublicSpaceResource,
+  buildShareSpaceResource,
+  isPublicSpaceResource
+} from 'web-client/src/helpers'
+import { configurationManager } from 'web-pkg'
+import { getCurrentInstance } from 'vue'
+import { useGettext } from 'vue3-gettext'
+
+export const useGetMatchingSpace = () => {
+  const store = useStore()
+  const configurationManager = useConfigurationManager()
+  const spaces = store.getters['runtime/spaces/spaces']
+  const { $gettext } = useGettext()
+  const getInternalSpace = (storageId) => {
+    return spaces.find((space) => space.id === storageId)
+  }
+
+  const getMatchingSpace = (resource: Resource): SpaceResource => {
+    if (resource.downloadURL.split('/')?.[5] === 'public-files') {
```

Comment:
This is not acceptable :-1:

Reply:
should not be needed anymore with my latest commit. can you check/confirm?

Reply:
was still needed, but we made it simpler together :heart: